app/views/sales/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Vendor Sales</h1>
+<ul>
+
+    <span><%=@sales.name%></span>
+    <span><%=@sales.vendor_id%></span>
+</ul>
```

Comment:
What are the `ul` tags doing? Without `li` bullet point lists they shouldn't do anything.